cmd/virt-handler/virt-handler.go

```diff
@@ -389,6 +415,12 @@ func (app *virtHandlerApp) Run() {
 		panic(fmt.Errorf("failed to detect the presence of selinux: %v", err))
 	}
 
+	capable, err := isNodeRealtimeCapable(logger)
+	if err != nil {
+		panic(fmt.Errorf("failed to identify if a node is capable of running realtime workloads: %v", err))
+	}
+	app.markNodeAsRealtimeCapable(capable, logger)
+
```

Comment:
@jordigilh That's a very good idea!
However, we should move this to the node-labeler. Virt-handler has a component where this functionality will fit perfectly.
Please take a look at [1]


[1] https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-handler/node-labeller/node_labeller.go#L241

Reply:
Code updated :smile:

Reply:
Perfect :)